src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ProducerConsumerStream.cs

```diff
@@ -190,9 +190,9 @@ private static void CancelAndDispose(object state)
             }
         }
 
-#pragma warning disable 660,661  // Hashcode not needed, Equals overriden for fast path comparison of EmptyContainer
+#pragma warning disable 659,660,661  // Hashcode not needed, Equals overriden for fast path comparison of EmptyContainer
         private struct WriteBufferWrapper : IEquatable<WriteBufferWrapper>
-#pragma warning restore 660,661
+#pragma warning restore 659,660,661
```

Comment:
I just re-read the comment by @iamjasonp and I'm confused why the disable on 228 didn't work. That was what Visual Studio complained about and VS created the quick fix for me. I wonder if we are running into problems of different compilers requiring the warning to be disabled in different places. I'm okay with having it all at the class level as it's not a large class.